deploy.ps1

```diff
@@ -68,11 +68,11 @@ $ErrorActionPreference = "Stop"
 
 # sign in
 Write-Host "Logging in...";
-Login-AzureRmAccount;
+Login-AzAccount;

```

Comment:
It should be Connect-AzAccount

Reply:
I agree... For Powershell, it's Connect-AzAccount